### ISSUE TITLE
Prompt string being returned as transcription result

### ISSUE THREAD

**Issue by nchudleigh** [argmaxinc/WhisperKit#162]
```swift
let promptString = "This will end up being returned as transcriptionResult"

let transcriptionResult = try await whisperKit.transcribe(
    audioArray: samples,
    decodeOptions: DecodingOptions(
        language: modeManager.activeMode.language,
        skipSpecialTokens: true,
        promptTokens: whisperKit.tokenizer.encode(text: promptString)
    )
)

print(promptString == transcriptionResult) // true
```

**Comment by ZachNagengast** [comment]
This is happening because the current promptTokens will have special tokens in them without filtering. I'm adding a fix that will filter it by default via `whisperKit.tokenizer.encode(text: promptString).filter { $0 < tokenizer.specialTokens.specialTokenBegin }`

Thanks for the report, this will be in the next release shortly.

**Comment by ZachNagengast** [comment]
Should be resolved with #183
